ui/src/scss/_patterns_table.scss

```diff
@@ -19,4 +19,5 @@ td:not(.p-table-expanding__panel) [class*="p-button"] {
 .u-align-icons--top {
   align-items: flex-start;
   display: flex;
+  padding-top: 0.3125rem;
```

Comment:
Ideally there should be an issue for this as well.

Reply:
That's fair. I'll move it overrides file with [this Vanilla issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/2504).